usr/src/uts/common/fs/zfs/zil.c

```diff
@@ -95,11 +95,10 @@ int zfs_commit_timeout_pct = 5;
 int zil_replay_disable = 0;
 
 /*
- * Tunable parameter for debugging or performance analysis.  Setting
- * zfs_nocacheflush will cause corruption on power loss if a volatile
- * out-of-order write cache is enabled.
+ * Disable the DKIOCFLUSHWRITECACHE commands that are normally sent to
+ * the disk(s) by the ZIL after an LWB write has completed.
```

Comment:
This also adds the risk of data loss, similar to zfs_nocacheflush.
I think the warning should remain.